lib/ansible/cli/galaxy.py

```diff
@@ -498,6 +498,10 @@ def add_install_options(self, parser, parents=None):
             install_parser.add_argument('--ignore-signature-status-code', dest='ignore_gpg_errors', type=str, action='append',
                                         help=ignore_gpg_status_help, default=C.GALAXY_IGNORE_INVALID_SIGNATURE_STATUS_CODES,
                                         choices=list(GPG_ERROR_MAP.keys()))
+            install_parser.add_argument('--offline', dest='offline', action='store_true', default=False,
+                                        help='Install collection artifacts (tarfiles) without contacting any distribution servers. '
+                                             'This does not apply to collections in git repositories. '
```

Comment:
When it's not a command, Git is usually capitalized:
```suggestion
                                             'This does not apply to collections in Git repositories. '
```

Reply:
Good catch. I might as well clarify remote URLs too... I used `--offline` since that's the option name for `verify` and role stuff but I wonder if that's confusing. Feel free to bikeshed. `--no-server`?
```suggestion
                                             'This does not apply to collections in Git repositories or prevent fetching remote URLs. '
```

Reply:
I do like `--offline`. Although, maybe something more specific like `--offline-artifacts` would do too.